server/static/js/js_tests/transcript/data/basic.txt

```diff
@@ -1,570 +1,107 @@
-
-SE 101      Introduction to Methods      0.25/0.25   99        Y      Y                   
-
-of Software Engineering                                                       
-
-                                                                        
-
-Term Avg                94.82  Term Units In Avg      2.75   Term Units Earned      2.75  
-
-Cumulative Avg          94.82  Total Units In Avg     2.75   Total Units Earned     2.75  
-
-Total Units Taken       2.75   Total Units Passed     2.75                                
-
-             Total Transfer Units   0.00                                
-
-                                                                        
-
-Academic Standing:       Excellent standing                                               
-
-Honours:                 Term Dean's Honours List                                         
-
-                                                                        
-
-__________________________________________________________________________________________
-
-                                                                        
-
-Non-Course Milestones                                                                     
-
-English Proficiency            Completed                     September 28, 2009           
-
-Technical Presentation         Completed                     August 25, 2011              
-
-Work Report 1                  Completed                     December 30, 2010            
-
-Work Report 2                  Completed                     August 30, 2011              
-
-Work Term 1                    Completed                     August 30, 2010              
-
-Work Term 2                    Completed                     April 30, 2011               
-
-Work Term 3                    Completed                     December 30, 2011            
-
-Workplace Hazardous Materials  Completed                     September 28, 2009           
-
-                                                                        
-
-Scholarships & Awards                                                                     
-
-2009-2010      Engineering Faculty/Staff Entrance Scholarship                             
-
-2009-2010      University of Waterloo President's Scholarship of Distinction              
-
-2010-2011      President's International Experience Award                                 
-
-2011-2012      President's Research Award                                                 
-
-                                                                        
-
-       - - - - -   End of Transcript   - - - - -                        
-
-
-
-
-
-
-Return
-
-
-
-
-
-My Academics	 	 	 	Pre-enrollment (undergrad Only)	 	 	 	Enroll	 	 
-
-
-My Undergrad Program	 	 	 	Grades	 	 	 	Unofficial Transcript	 	 
-
-
-
-
-
-
-
-
-
-
-
-
-
-Vimium has been updated to 1.39.x
+University of Waterloo
```

Comment:
Haha the replacement of these fixtures is wonderfully symbolic of the ownership transfer